packages/commons/src/big-query/liquidity-indices-table/pull-data/pullClosestIndices.ts

```diff
@@ -0,0 +1,54 @@
+import { getBigQuery } from '../../client';
+import { getTableFullName } from '../../utils/getTableName';
+import { TableType } from '../../types';
+import { Address } from '../../../utils/convertLowercase';
+
+export type LiquidityIndexEntry = {
```

Comment:
we already have this. https://github.com/Voltz-Protocol/v2-off-chain-monorepo/blob/87c68858c55f899f5aa14e41d79a73e2e4e4d858/packages/commons/src/big-query/liquidity-indices-table/types.ts#L3

Reply:
oh thanks!! replaced [e0c8edf](https://github.com/Voltz-Protocol/v2-off-chain-monorepo/pull/34/commits/e0c8edf872ac0156572fc067bdb129d70fc36e1d)